notty-community.opam

```
@@ -1,17 +1,21 @@
-# This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
 synopsis: "Declaring terminals"
-description:
-  "Notty is a declarative terminal library for OCaml structured around a notion of composable images. It tries to abstract away the basic terminal programming model, providing something simpler and more expressive."
+description: """
+Notty is a declarative terminal library for OCaml structured around a notion
+of composable images. It tries to abstract away the basic terminal
+programming model, providing something simpler and more expressive.
+
+This is a branch from the original notty, maintained by the community.
+"""
 maintainer: [
   "Ali Caglayan <alizter@gmail.com>"
   "Antonin Décimo <antonin@tarides.com>"
```

```suggestion
  "Shon Feder <shon.feder@gmail.com>"
```
if i'm understanding correctly? @Alizter do you want to be removed too or are you fine maintaining it?

---

I'm fine with staying as a maintainer.